datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/ResolverUtils.java

```diff
@@ -56,6 +56,14 @@ public static Authentication getAuthentication(DataFetchingEnvironment environme
         return ((QueryContext) environment.getContext()).getAuthentication();
     }
 
+    /**
+     * @apiNote DO NOT use this method if the facet filters do not include `.keyword` suffix to ensure
```

Comment:
Is there another method that will do this , while _also_ injecting the ".keyword" part?

Reply:
Not exactly, you have ResolverUtils#buildFilter which takes `@Nullable List<FacetFilterInput> facetFilterInputs` as input instead of `@Nullable List<FacetFilterInput> facetFilterInputs, @Nonnull Set<String> validFacetFields`